src/rdo.rs

```diff
@@ -159,13 +159,12 @@ pub fn get_lambda_sqrt(fi: &FrameInvariants) -> f64 {
   q0 * (std::f64::consts::LN_2 / 6.0).sqrt()
 }
 
-// Compute the rate-distortion cost for an encode
-fn compute_rd_cost(
+// Compute the distortion cost for an encode
```

Comment:
We should only use the word "cost" to refer to the RD cost. "for an encode" in this comment doesn't mean anything, so no comment at all would be better. (And even better would be a comment explaining what all the parameters of this function do).

Reply:
Yes. I renamed some of them to rate but missed some.